packages/@azure/servicebus/data-plane/test/testUtils.ts

```diff
@@ -82,13 +121,50 @@ export function getSenderClient(
   throw new Error("Cannot create sender client for give client type");
 }
 
-export function getReceiverClient(
+export async function getReceiverClient(
   namespace: Namespace,
   clientType: ClientType
-): QueueClient | SubscriptionClient {
+): Promise<QueueClient | SubscriptionClient> {
   switch (clientType) {
     case ClientType.PartitionedQueue:
-      return namespace.createQueueClient(process.env.QUEUE_NAME || "partitioned-queue");
+      const subscriptionId = process.env["AZURE_SUBSCRIPTION_ID"] || "";
```

Comment:
It looks like there is a bunch of duplicated code here -- can we pull some of this code in a common method?

Reply:
Yes, I believe this was a prototype for which Harsha needed some feedback before moving forward.

We will be moving this out to a function and also call it from inside each `case` statement

Reply:
Yes, I wanted some feedback before going forward.
As Ramya said, I'll be creating separate functions to create and delete the queues/topics/subscriptions and will be using them.